src/06-extensions.md

```diff
@@ -38,7 +38,6 @@ provides a direct link to the documentation.
 | [BEP009](https://docs.google.com/document/d/1mqMLnxVdLwZjDd4ZiWFqjEAmOmfcModA_R535v3eQs0) | Positron Emission Tomography (PET)                                                                         | Melanie Ganz                                                               |
 | [BEP011](https://docs.google.com/document/d/1YG2g4UkEio4t_STIBOqYOwneLEs1emHIXbGKynx7V0Y) | The structural preprocessing derivatives                                                                   | Andrew Hoopes                                                              |
 | [BEP012](https://docs.google.com/document/d/16CvBwVMAs0IMhdoKmlmcm3W8254dQmNARo-7HhE-lJU) | The functional preprocessing derivatives                                                                   | Camille Maumet and Chris Markiewicz                                        |
-| [BEP013](https://docs.google.com/document/d/1qBNQimDx6CuvHjbDvuFyBIrf2WRFUOJ-u50canWjjaw) | The resting state fMRI derivatives                                                                         | Steven Giavasis                                                            |
 | [BEP014](https://docs.google.com/document/d/11gCzXOPUbYyuQx8fErtMO9tnOKC3kTWiL9axWkkILNE) | The affine transformations and nonlinear field warps                                                       | Oscar Esteban                                                              |
 | [BEP015](https://docs.google.com/document/d/1WYOTXDB7GzlHoWqLjd45I3uGBgPxXddST-NTqBnroJE) | Mapping file                                                                                               | Eric Earl, Camille Maumet, and Vasudev Raguram                             |
 | [BEP016](https://docs.google.com/document/d/1cQYBvToU7tUEtWMLMwXUCB_T8gebCotE1OczUpMYW60) | The diffusion weighted imaging derivatives                                                                 | Franco Pestilli and Oscar Esteban                                          |
```

Comment:
BEPs 12, 13 and 16 are now #207.

Reply:
alright, I'll update it and leave a note on the respective Google Docs

Reply:
BEP018 - I just need to make a markdown -- this was agreed a while ago with @chrisgorgo but we never did it

Reply:
what are you referring to @CPernet? BEP018 is listed here: https://bids-specification.readthedocs.io/en/stable/06-extensions.html#bids-extension-proposals

... and I assume it's work in progress still?

Reply:
nope it's actually finished - months ago (but I was stuck on some EEG stuff ... )

Reply:
I see, cool :-) ... let's wait for your markdown PR then